src/views/auth/SignUp.jsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import {
+  Avatar,
+  Button,
+  TextField,
+  FormControlLabel,
+  Checkbox,
+  Link,
+  Grid,
+  Box,
+  Typography,
+  Container,
+} from '@mui/material';
+
+import CssBaseline from '@mui/material/CssBaseline';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { Copyright } from 'components';
+
+const defaultTheme = createTheme();
+
+export function SignUp() {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+    console.log({
+      email: data.get('email'),
+      password: data.get('password'),
+    });
+  };
+
+  return (
+    <ThemeProvider theme={defaultTheme}>
+      <Container component="main" maxWidth="xs">
+        <CssBaseline />
```

Comment:
<CssBaseline/> can be imported once per  page/project... Please, remove 15, 17, 20, 33, 35, 120 lines. Already pesent in Layout.jsx

Reply:
lines removed, created new commit